### PR TITLE
the task is created

### PR DESCRIPTION
**Pull request by Anastasiya145**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
```
Comment (vadiimvooo) [inline client/public/index.html:16]: Remove redundant comments.

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (vadiimvooo) [inline client/public/index.html:27]: Change title for the app

File: client/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (vadiimvooo) [inline client/public/index.html:30]: This can be also removed.

File: client/src/pages/SignUp.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    },
+    validationSchema: Yup.object().shape({
+      email: Yup.string()
+        .email('Invalid email address')
+        .required('Email is required'),
+      password: Yup.string()
+        // .min(8, 'Password must be at least 8 characters')
+        // .matches(/[A-Z]/, 'Password must contain at least one uppercase letter')
+        // .matches(/[a-z]/, 'Password must contain at least one lowercase letter')
+        // .matches(/[0-9]/, 'Password must contain at least one number')
```
Comment (vadiimvooo) [inline client/src/pages/SignUp.tsx:37]: Remove comments.

File: client/src/pages/SignIn.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    },
+    validationSchema: Yup.object().shape({
+      email: Yup.string()
+        .email('Invalid email address')
+        .required('Email is required'),
+      password: Yup.string()
+        // .min(8, 'Password must be at least 8 characters')
+        // .matches(/[A-Z]/, 'Password must contain at least one uppercase letter')
+        // .matches(/[a-z]/, 'Password must contain at least one lowercase letter')
+        // .matches(/[0-9]/, 'Password must contain at least one number')
```
Comment (vadiimvooo) [inline client/src/pages/SignIn.tsx:39]: Remove comments

File: client/src/pages/ResetPassword.tsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+import CustomTextField from "../components/CustomTextField";
+
+const ResetPassword: FC = () => {
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    console.log({
+      email: data.get("email"),
+      password: data.get("password"),
+    });
```
Comment (vadiimvooo) [inline client/src/pages/ResetPassword.tsx:19]: This should be removed.

File: client/src/pages/ResetPassword.tsx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  const navigate = useNavigate();
+
+  return (
+    <LoginLayout>
+      <>
+        <Typography component="h1" variant="h5" sx={{ textAlign: "center" }}>
+          Recover your password or username
+        </Typography>
+        <Box component="form" onSubmit={handleSubmit} noValidate sx={{ mt: 1 }}>
+          {/* <CustomTextField label="Email Address" field="email" formik={undefined} /> */}
```
Comment (vadiimvooo) [inline client/src/pages/ResetPassword.tsx:31]: Remove comments.

File: client/src/pages/SignIn.tsx [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+        .required('Password is required'),
+    }),
+    onSubmit: (values) => {
+      console.log(values);
```
Comment (vadiimvooo) [inline client/src/pages/SignIn.tsx:43]: Remove console.log

File: client/src/pages/SignUp.tsx [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+        .required('Password is required'),
+    }),
+    onSubmit: (values) => {
+      console.log(values);
```
Comment (vadiimvooo) [inline client/src/pages/SignUp.tsx:41]: Remove console.log

File: client/src/pages/Users.tsx [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+const Users: FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+
+  const fetchUsers = async () => {
+    try {
+      const response = await userService.getAll();
+      const users: User[] = response.data;
+      setUsers(users);
+    } catch (error) {
+      console.log('Error fetching users:', error);
```
Comment (vadiimvooo) [inline client/src/pages/Users.tsx:15]: Try to change this log for some popup like toastr.

File: server/src/index.js [hunk 11]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+require('dotenv/config');
+const express = require('express');
+const cors = require('cors');
+const authRouter = require('./routes/auth.routes');
+const userRouter = require('./routes/user.routes');
+
+console.log(process.env.PORT);
```
Comment (vadiimvooo) [inline server/src/index.js:9]: Remove log

File: server/src/routes/auth.routes.js [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+authRouter.get('/', authControler.getAll);
+authRouter.post(
+  '/registration',
+  authMiddleware.validateEmailAndPasswordReqParams,
+  authControler.register,
+);
+authRouter.post('/activation', (req, res) => res.send('activation'));
+authRouter.post(
+  '/login',
+  // authMiddleware.validateEmailAndPasswordReqParams,
```
Comment (vadiimvooo) [inline server/src/routes/auth.routes.js:16]: Remove this as well

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Good job. A few changes required.

File: client/src/pages/SignUp.tsx [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+    },
+    validationSchema: Yup.object().shape({
+      email: Yup.string()
+        .email("Invalid email address")
+        .required("Email is required"),
+      password: Yup.string()
+        .min(6, "Password must be at least 6 characters")
+        // .matches(/[A-Z]/, 'Password must contain at least one uppercase letter')
+        // .matches(/[a-z]/, 'Password must contain at least one lowercase letter')
+        // .matches(/[0-9]/, 'Password must contain at least one number')
```
Comment (lerastarynets) [inline client/src/pages/SignUp.tsx:38]: pls remove redundant comments

**Review by lerastarynets — APPROVED** [review]
lgtm 🚀

**Review by Anastasiya145 — COMMENTED** [review]
Please review my code (the last two commits)
I added the code for autogeneration of swagger doc
the API doc is here - https://app.swaggerhub.com/apis/asiva/auth-users_rest_api/1.0.0 
can you please check is doc looks correctly and good as real project? I want to add it later to my CV
thanks!

**Pull request closed without merging**